tsconfig.json

```diff
@@ -1,11 +1,18 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "es5",
+    "strict": true,
```

Comment:
`"strict": true` is a good change, but conceptually it's independent from setting up injection. Would you be OK splitting it into a separate PR?